tests/ttnn/module.mk

```diff
@@ -0,0 +1,7 @@
+include $(TT_METAL_HOME)/tests/ttnn/unit_tests/module.mk
+
+tests/ttnn: tests/ttnn/unit_tests_run
```

Comment:
Hmm... we don't actually have any commands in Make to run tests. Only build

I'm open to adding them, but `tests/ttnn` is a build command as you put in `tests/module.mk`. I think what you could do is instead:

```
# Only builds the tests, and specifically tests/ttnn/unit_tests in tests/ttnn/unit_tests/module.mk 
tests/ttnn: tests/ttnn/unit_tests

# although I would probably move this directive into the tests/ttnn/unit_tests/module.mk file because it's specifically for unit tests
.PHONY: tests/ttnn unit_tests_run
tests/ttnn/unit_tests_run: tests/ttnn/unit_tests
	@echo "Running all ttnn unit tests"
```

`tests/ttnn/unit_tests` would be a new directive in `tests/ttnn/unit_tests/module.mk` to specifically target your unit tests.

Reply:
Updated tests/ttnn/unit_tests_run to run the tests as well.